tests/models/llama/test_modeling_llama.py

```diff
@@ -735,3 +736,138 @@ def test_model_7b_logits(self):
         ]
         infilling = tokenizer.batch_decode(generated_ids)
         self.assertEqual(infilling, EXPECTED_INFILLING)
+
+
+@slow
```

Comment:
This set of slow tests was now moved to the llama test file -> if we run the slow llama tests, this will now be triggered